x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/counted_keyword/10_basic.yml

```diff
@@ -0,0 +1,53 @@
+setup:
+
+  - skip:
+      version: " - 8.11.99"
+      reason: "counted_keyword was added in 8.12"
+
+  - do:
+      indices.create:
+        index: test-events
+        body:
+          settings:
+            number_of_shards: 1
```

Comment:
One shard is the default. Let's remove the settings from this create index api call?